src/Datadog.Trace/Logging/LibLogScopeEventSubscriber.cs

```diff
@@ -106,15 +112,15 @@ public void Dispose()
 
         private void SetDefaultValues()
         {
-            SetCorrelationIdentifierContext(0, 0);
+            SetCorrelationIdentifierContext(_defaultServiceName, _version, _env, 0, 0);
         }
 
         private void RemoveLastCorrelationIdentifierContext()
         {
             // TODO: Debug logs
-            for (int i = 0; i < 2; i++)
+            for (int i = 0; i < 5; i++)
```

Comment:
Maybe add a comment as to what the magic number (5 in this case) means, or use a symbolic name for it.

Reply:
Good call, added a variable in f771da4f6f788270198bf3a034eb65dacf7f7a94